cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,24 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+const logUsername = 'dsarvaaav';
+const email = 'fdsfaasd13as@gmail.com';
+const password = 'dsadfgafsa';
 
+describe('my firs test', () => {
+  it('should', () => {
+    cy.visit('https://conduit.mate.academy/');
+
+    cy.get(':nth-child(3) > .nav-link').click();
+
+    cy.get(':nth-child(1) > .form-control')
+      .type(logUsername);
+
+    cy.get(':nth-child(2) > .form-control')
+      .type(email);
```

Comment:
The selector ':nth-child(2) > .form-control' is too specific. Consider using a more stable selector to prevent future issues if the form structure changes.